ios/Plugin/Plugin.swift

```diff
@@ -422,17 +423,20 @@ public class StripePlugin: CAPPlugin {
         ]
 
         let ctx = STPCustomerContext(keyProvider: self)
-        let pCfg = STPPaymentConfiguration.shared()
+        let pCfg = STPPaymentConfiguration.shared
 
         if let po = call.getObject("paymentOptions") as? [String: Bool] {
             if po["applePay"] ?? false {
-                pCfg.additionalPaymentOptions.insert(.applePay)
+                pCfg.applePayEnabled = true
             }
             if po["fpx"] ?? false {
-                pCfg.additionalPaymentOptions.insert(.FPX)
+                pCfg.fpxEnabled = true
             }
             if po["default"] ?? false {
-                pCfg.additionalPaymentOptions.insert(.default)
+                // TODO The additionalPaymentOptions is no longer an object, and has been deprecated
```

Comment:
AFAIK this function is hidden from documentation and it's not ready yet.

Used this to try and implement prebuilt UI features.

https://stripe.com/docs/mobile/ios/basic